packages/typespec-test/package.json

```diff
@@ -4,18 +4,18 @@
   "type": "module",
   "dependencies": {
     "@azure-tools/typespec-ts": "workspace:^0.38.6",
-    "@typespec/openapi": "^1.0.0-0",
-    "@azure-tools/typespec-autorest": ">=0.54.0 <1.0.0",
-    "@typespec/openapi3": "^1.0.0-0",
-    "@azure-tools/typespec-azure-core": ">=0.54.0 <1.0.0",
-    "@azure-tools/typespec-client-generator-core": ">=0.54.0 <1.0.0",
-    "@azure-tools/typespec-azure-resource-manager": ">=0.54.0 <1.0.0",
-    "@azure-tools/typespec-azure-rulesets": ">=0.54.0 <1.0.0",
-    "@typespec/compiler": "^1.0.0-0",
-    "@typespec/http": "^1.0.0-0",
-    "@typespec/rest": ">=0.68.0 <1.0.0",
-    "@typespec/versioning": ">=0.68.0 <1.0.0",
-    "@typespec/json-schema": "^1.0.0-0",
+    "@typespec/openapi": "1.0.0-rc.1",
+    "@azure-tools/typespec-autorest": ">=0.55.0 <1.0.0",
+    "@typespec/openapi3": "1.0.0-rc.1",
+    "@azure-tools/typespec-azure-core": ">=0.55.0 <1.0.0",
+    "@azure-tools/typespec-client-generator-core": ">=0.55.0 <1.0.0",
```

Comment:
should be latest dev: https://www.npmjs.com/package/@azure-tools/typespec-client-generator-core/v/0.56.0-dev.4